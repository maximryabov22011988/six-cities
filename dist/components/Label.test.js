'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Label_1 = require('./Label');
it('Label correctly render', () => {
  const tree = renderer
    .create(React.createElement(Label_1.default, { isShow: true, name: 'Premium', parentClassName: 'place-card' }))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Label.test.js.map
