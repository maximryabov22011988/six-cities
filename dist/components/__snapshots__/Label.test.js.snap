// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly with mandatory props 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
